parquet/src/main/java/org/apache/iceberg/parquet/ParquetMetrics.java

```diff
@@ -107,6 +109,14 @@ public static Metrics fromMetadata(ParquetMetadata metadata) {
         toBufferMap(fileSchema, lowerBounds), toBufferMap(fileSchema, upperBounds));
   }
 
+  public static List<Long> getOffsetRanges(ParquetMetadata md) {
```

Comment:
Is there a more appropriate file? Maybe this one should be renamed to `ParquetUtil`?